_includes/quick-start-module.js

```diff
@@ -115,9 +124,22 @@ function disableUnsupportedPlatforms(os) {
   });
 }
 
+function disableUnsupportedPackages(pkg) {
```

Comment:
Hmm, perhaps rather than say that conda is not supported, we just add installation instruction that uses conda-forge and mention that this is beta?